### DISCUSSION TITLE
한진 택배 403 에러 관련

### DISCUSSION THREAD

**Discussion by seungholee-dev** [shlee322/delivery-tracker#175] [General]
@shlee322 
안녕하세요, 상혁님 :) 

저희가 한진 택배 서비스 크롤러를 구현하던 도중 궁금한 게 있어서 남깁니다.
한진 택배사에 초당 1개씩 송장번호 조회를 요청하면 130~150개에서 403에러가 뜨는데요, 

혹시 이 부분 관련해서, delivery-tracker는 어떻게 해결하셨는지 궁금합니다!

감사합니다. 😄

**Comment by shlee322** [comment]
안녕하세요.

딜리버리트래커 클라우드 버전의 경우 오픈소스로 제공하는 영역 외에 안정적인 서비스 운영과 사용자가 경험하는 퀄리티 향상을 위해 다양한 방법을 사용하고 있습니다. 이에는 자체 우선순위 판단과 이에 따른 스크래핑 스케쥴링, 캐싱, QoS 등 방법등이 존재합니다.

이 또한 현재 제공하고 있는 v1 에서는 개선할 포인트들이 상당수 존재하여 v2를 개발중이며 일부 사용자를 대상으로 베타테스트 이후에 공개될 예정 입니다. 참여를 원하시면 contact@tracker.delivery 으로 연락 부탁드립니다.

감사합니다.

**Reply by shlee322** [reply]
v2 오픈으로 close 합니다.
